pkg/cmd/server/certs/create_kubeconfig.go

```diff
@@ -0,0 +1,137 @@
+package certs
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd"
+	clientcmdapi "github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd/api"
+)
+
+type CreateKubeConfigOptions struct {
+	APIServerURL       string
+	PublicAPIServerURL string
+	APIServerCAFile    string
+	ServerNick         string
+
+	CertFile string
+	KeyFile  string
+	UserNick string
+
+	KubeConfigFile string
+}
+
+func NewCommandCreateKubeConfig() *cobra.Command {
+	options := &CreateKubeConfigOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "create-kubeconfig",
+		Short: "Create a basic .kubeconfig file from client certs",
```

Comment:
Add a description about what the output is (e.g. current context named "<--cluster>" pointing to --master host, optional "public-<--cluster>" context pointing to --public-master host if specified)